db/fixtures/server_roles.csv

```diff
@@ -20,5 +20,6 @@ storage_metrics_collector,Storage Capacity & Utilization Data Collector,0,false,
 storage_metrics_coordinator,Storage Capacity & Utilization Coordinator,1,false,zone
 storage_metrics_processor,Storage Capacity & Utilization Data Processor,0,false,zone
 user_interface,User Interface,0,false,region
+websocket,Websocket,0,false,region
```

Comment:
alphabetically, this should come after vmdb_storage_bridge (or perhaps at the end)